simtools/simtel/simtel_histograms.py

```diff
@@ -97,7 +97,7 @@ def _combineHistogramFiles(self):
             countFile = True
             with EventIOFile(file) as f:
 
-                for i, o in enumerate(yield_toplevel_of_type(f, Histograms)):
+                for _, o in enumerate(yield_toplevel_of_type(f, Histograms)):
```

Comment:
We can remove the enumerate if the `i` is not used anywhere

Reply:
OK. 35c61af